src/main/java/millom/sandbox/pojos/Descriptions.java

```diff
@@ -0,0 +1,287 @@
+package millom.sandbox.pojos;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Descriptions {
+  @JsonProperty("disclaimer_en")
+  private String disclaimerEn;
+
+  @JsonProperty("disclaimer_es")
+  private String disclaimerEs;
+
+  @JsonProperty("sol_desc_en")
+  private String sol_desc_en;
+
+  @JsonProperty("sol_desc_es")
+  private String sol_desc_es;
+
+  @JsonProperty("terrestrial_date_desc_en")
+  private String terrestrialDateDescEn;
+
+  @JsonProperty("terrestrial_date_desc_es")
+  private String terrestrialDateDescEs;
+
+  @JsonProperty("temp_desc_en")
+  private String tempDescEn;
+
+  @JsonProperty("temp_desc_es")
+  private String tempDescEs;
+
+  @JsonProperty("pressure_desc_en")
+  private String pressureDescEn;
+
+  @JsonProperty("pressure_desc_es")
+  private String pressureDescEs;
+
+  @JsonProperty("abs_humidity_desc_en")
+  private String absHumidityDescEn;
+
+  @JsonProperty("abs_humidity_desc_es")
+  private String absHumidityDescEs;
+
+  @JsonProperty("wind_desc_en")
+  private String windDescEn;
+
+  @JsonProperty("wind_desc_es")
+  private String windDescEs;
+
+  @JsonProperty("gts_temp_desc_en")
+  private String gtsTempDescEn;
+
+  @JsonProperty("gts_temp_desc_es")
+  private String gtsTempDescEs;
+
+  @JsonProperty("local_uv_irradiance_index_desc_en")
+  private String localUvIrradianceIndexDescEn;
+
+  @JsonProperty("local_uv_irradiance_index_desc_es")
+  private String localUvIrradianceIndexDescEs;
+
+  @JsonProperty("atmo_opacity_desc_en")
+  private String atmoOpacityDescEn;
+
+  @JsonProperty("atmo_opacity_desc_es")
+  private String atmoOpacityDescEs;
+
+  @JsonProperty("ls_desc_en")
+  private String lsDescEn;
+
+  @JsonProperty("ls_desc_es")
+  private String lsDescEs;
+
+  @JsonProperty("season_desc_en")
+  private String seasonDescEn;
+
+  @JsonProperty("season_desc_es")
+  private String seasonDescEs;
+
+  @JsonProperty("sunrise_sunset_desc_en")
+  private String sunriseSunsetDescEn;
+
+  @JsonProperty("sunrise_sunset_desc_es")
+  private String sunriseSunsetDescEs;
+
+  private Descriptions(){
+  }
+
+  public Descriptions(DescriptionsBuilder descriptionsBuilder) {
```

Comment:
Can it be private?

Reply:
changed to private

` private Descriptions(DescriptionsBuilder descriptionsBuilder) {`